src/biogeophys/SoilTemperatureMod.F90

```diff
@@ -47,7 +47,7 @@ module SoilTemperatureMod
   !     o The thermal conductivity of soil is computed from
   !       the algorithm of Johansen (as reported by Farouki 1981), and the
   !       conductivity of snow is from the formulation used in
-  !       SNTHERM (Jordan 1991).
+  !       Sturm (1997).
```

Comment:
Why is there so much documentation in the code itself here?  If we keep all the notes  (here and below) snow conductivity should reference both Jordan and Sturm papers.

Reply:
Half of the notes here and below are about the thermal conductivity of soil, so I will keep them and reference both papers.